SpatialGDK/Source/SpatialGDKEditor/Private/SchemaGenerator/TypeStructure.cpp

```diff
@@ -12,13 +12,24 @@ using namespace SpatialGDKEditor::Schema;
 
 TArray<EReplicatedPropertyGroup> GetAllReplicatedPropertyGroups()
 {
-	static TArray<EReplicatedPropertyGroup> Groups = { REP_MultiClient, REP_SingleClient };
+	static TArray<EReplicatedPropertyGroup> Groups = { REP_MultiClient, REP_SingleClient, REP_InitialOnly };
```

Comment:
Nit: Could change this to just return { ... }, copying from the heap arguably more expensive than from the stack

Reply:
Oh interesting - static version would only be constructed the once though - whereas stack version would be everytime. I guess stack version would benefit from RVO though...

Reply:
Yeah, without RVO you'd still be better off (TArray has an initializer_list constructor instead of using the TArray& copy) - but good point RVO would improve things even further!